src/Inspection.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+
+namespace RicorocksDigitalAgency\Morpher;
+
+
+class Inspection
+{
+    public $beforeHook;
+    public $afterHook;
+
+    public function before(callable $closure)
+    {
+        $this->beforeHook = $closure;
+        return $this;
+    }
+
+    public function after(callable $closure)
+    {
+        $this->afterHook = $closure;
```

Comment:
Could we use `tap` for these assignments?

Reply:
Changed.